### DISCUSSION TITLE
Need to reset production DB, how to do it?

### DISCUSSION THREAD

**Discussion by gmcamposano** [prisma/prisma#19250] [Q&A] [answered]
I made a change that adds a new column with no default value, which ends up throwing an error whenever i want to build in my pipeline. Ex: There is a new computed column named TOTAL which did not exist in the past migration and affects the database as values are going to be empty:
ERROR: column "total" of relation "Numero_de_factura" contains null values

What can i do?

**Comment by ludralph** [comment]
Hi @gmcamposano 👋 

Can you take a look at this [section](https://www.prisma.io/docs/guides/migrate/developing-with-prisma-migrate/generating-down-migrations#:~:text=If%20your%20up%20migration%20was,with%20the%20migrate%20dev%20command.) of the documentation on how you can revert the change that is causing the error.

**Reply by gmcamposano** [reply]
Yes, I saw it. But i do not want to revert it, there is no way to revert that change as its a breaking change.

**Reply by ludralph** [reply]
Would creating a migration that takes your production database to the state of your data model before the last migration help as seen in this [section](https://www.prisma.io/docs/guides/migrate/production-troubleshooting#moving-backwards-and-reverting-all-changes)?

**Reply by gmcamposano** [reply]
No it wouldn't as its a column that was added that has no default value. That means all other values for that row will be null.

**Reply by ludralph** [reply]
Hi @gmcamposano 

Can you please clarify how you introduced the new column with no default value to the DB? Is it through the database directly or through migrations in Prisma

**Reply by gmcamposano** [reply, marked as answer]
I solved it by manually deleting the production database and starting again. It was a fairly new DB and most of data were only for testing in production environment.
